packages/idyll-document/src/runtime.js

```diff
@@ -176,8 +176,7 @@ const createWrapper = ({ theme, layout, authorView }) => {
         })
       });
       const metaData = childComponent.type._idyll;
-      // give authorView default false value
-      if (this.isAuthorView && metaData && metaData.props) {
+      if (this.isAuthorView != 'false' && metaData && metaData.props) {
```

Comment:
always use strict equality checks `===` and `!==` unless there is some good reason for doing otherwise